pages/home.md

```diff
@@ -88,14 +84,14 @@ permalink: /
           </div>
           <div class="column is-offset-1 is-5">
             <h2 class="title">
-              Using interoperable data standards
+              Linked Data: Interoperable data standards
```

Comment:
I just wanted a little continuation from the other section where we introduce Linked Data.

Also, by sheer chance, this makes all the left hand titles be active verb phrases:

- Store anything
- Share it safely

and the left hand side, subject introductions:

- Pods: ...
- Linked Data: ...

Reply:
Heh - per later conversation, will remove reference to Linked Data.
Further discussion can be made on that patch.